Contribute/index.md

```diff
@@ -0,0 +1,140 @@
+# Contribute to Umbraco Documentation
+
+This guide will cover the following sections:
+ - [Ways of contributing](#ways-of-contributing)
+ - [Markdown conventions](#markdown-conventions)
+ - [Contribution badge](#contribution-badge)
+ - [Documentation Curators](#documentation-curators)
+
+ ## Ways of contributing
+This guide will cover how to contribute to the Umbraco Documentation, if you are interested in contributing to the Umbraco CMS you can find out how in their [contribution guidelines](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/.github/CONTRIBUTING.md).
+
+There are two ways to contribute to the Umbraco Documentation:
+- [Creating an issue](#creating-an-issue)
+- [Creating a pull request](#creating-a-pull-request)
+
+### Creating an issue
+The Umbraco Documentation uses [Github issues]((https://github.com/umbraco/UmbracoDocs/issues)) to manage issues with the documentation.
+You are very welcome to create an issue if you feel some documentation is missing or wrong, the issue board is also used for discussions about possible improvements or ways to deal with something in the documentation. You can also find a button in the top right corner of every page in the documentation itself that looks like this:
```

Comment:
We should use issues only in the following cases:

1. **If the issue will need more than one PR.**
2. if you feel some documentation is wrong or missing and you do not have time or knowledge to do a PR
3. open a discussion about possible improvements or ways to deal with something in the documentation